core/src/test/scala/higherkindness/mu/rpc/srcgen/AvroSrcGenTests.scala

```diff
@@ -28,36 +31,32 @@ import org.scalatest._
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpec
 import org.scalatestplus.scalacheck.Checkers
-import scala.io._
 
 class AvroSrcGenTests extends AnyWordSpec with Matchers with OneInstancePerTest with Checkers {
 
   "Avro Scala Generator" should {
 
     "generate correct Scala classes" in {
       check {
-        forAll { scenario: Scenario => test(scenario) }
+        forAll { scenario: Scenario =>
+          test(scenario)
+        }
       }
     }
 
     "return a non-empty list of errors instead of generating code from an invalid IDL file" in {
-      val response = {
-        AvroSrcGenerator(
-          List(BigDecimalAvroMarshallers),
-          ScalaBigDecimalTaggedGen,
-          CompressionType.Identity
-        ).generateFrom(
-          Source.fromInputStream(getClass.getResourceAsStream("/avro/Invalid.avdl")).mkString,
+      val actual :: Nil = {
+        AvroSrcGenerator(CompressionType.Identity, MonixObservable, true).generateFrom(
+          Set(new File(getClass.getResource("/avro/Invalid.avdl").toURI)),
           Avro
         )
       }
 
-      response shouldBe Some(
-        (
-          "foo/bar/MyGreeterService.scala",
-          Validated.invalidNel(
-            "RPC method response parameter has non-record response type 'STRING'"
-          )
+      actual.inputFile.getPath should endWith("/avro/Invalid.avdl")
+
+      actual.output shouldEqual Invalid(
+        NonEmptyList.one(
+          "Encountered an unsupported response type: Skeuomorph only supports Record types for Avro responses. Encountered response schema with type STRING"
```

Comment:
Should we add this error message to the mu documentation, e.g. on [this line](https://github.com/higherkindness/mu-scala/pull/1028/files#diff-8c5db881bab90b64b85764aa5562395206944cbb8cbcd4efb15ec59ecdbd8489R184)?  I know that we do say that we don't support non-record types for Avro but some folks tend to google error messages blindly (even if it's a good error message that says exactly what the problem is, like this one), and so it might be nice for our SEO to have an error message like this appear in our documentation.  I'm not married to this idea fyi, just kinda noodling on it.  Open to adding it or leaving it as it is, just figured I'd ask.

Reply:
Yes I changed this error message from the one in the docs. Will update my doc PR

Reply:
👍

Reply:
added here https://github.com/higherkindness/mu-scala/pull/1028/files#diff-8c5db881bab90b64b85764aa5562395206944cbb8cbcd4efb15ec59ecdbd8489R150-R151